superglue/lib/action_creators/requests.ts

```diff
@@ -152,6 +152,7 @@ export function visit(
     placeholderKey,
     beforeSave = (prevPage, receivedPage) => receivedPage,
     revisit = false,
+    action = 'push',
```

Comment:
Since we want to check for `undefined` later in this method, we do have to start out as: 

```suggestion
    action,
```